ERCS/erc-interoperable-delegated-account.md

```diff
@@ -0,0 +1,172 @@
+---
+eip: <to be assigned>
+title: Interoperable Delegated Accounts
+description: Interoperable Delegated Account interfaces and behaviors for better interoperability and storage management.
+author: David Kim (@PowerStream3604), Richard Meissner (@rmeissner), Akshay Patel (akshay-ap), Joshua Kim (@JoshuaKim3604), Fangting (@trinity-0111)
+discussions-to: https://ethereum-magicians.org/t/erc-interoperable-delegated-account/21237
```

Comment:
```suggestion
discussions-to: https://ethereum-magicians.org/t/erc-7779-interoperable-delegated-account/21237
```
I added the ERC number to Eth Magicians topic title

Reply:
I've reflected the updated link in the follow up commit